integration/testdata/tstestrunner/vitest.config.ts

```diff
@@ -8,7 +8,8 @@ export default defineConfig({
   test: {
     include: ['**/*_test.ts'],
     globals: true,
-    setupFiles: 'testutil/setup.ts'
+    setupFiles: 'testutil/setup.ts',
+    testTimeout: 10000
```

Comment:
The timeout from the tests could be removed in favor of using these 10000ms which will be applied to all tests.